docs/managed_2gp.rst

```diff
@@ -0,0 +1,162 @@
+
+
+Create a Beta Version
+---------------------
+
+CumulusCI uses the ``dependencies`` section of your ``cumulusci.yml`` file to define your 2GP project's dependencies.
+CumulusCI uses GitHub releases to identify the ancestor id and new version number for the beta package version. By default,
+the new beta version will increment the minor version number from the most recent GitHub release.
+
+Because Salesforce requires package version Ids (``04txxxxxxxxxxxx``) for 2GP package dependencies, some CumulusCI
+dependencies must be installed in an org to make those Ids available. If your project has dependencies that are not
+specified as a ``version_id``, start by running
+
+.. code-block:: console
+
+    $ cci flow run dependencies --org dev
+
+When you're ready, and your org is prepared, to upload a package version, run the command
+
+.. code-block:: console
+
+    $ cci flow run upload_2gp_beta --org dev
+
+.. important::
+    
+    The org supplied to ``upload_2gp_beta`` has two purposes. One is to look up the Ids of dependency packages (see above).
+    The other is to provide the configuration for the *build org* used to upload the 2GP package version. CumulusCI will use
+    the scratch org definition file used to create the specified org (``dev`` here) to create the build org, which defines
+    the features and settings available during package upload.
+
+    You may wish to define a separate scratch org configuration just for package uploads to ensure only your required features
+    are present.
+
+
+The ``upload_2gp_beta`` flow executes these tasks:
+
+* Uploads a new beta version of the managed package.
+* Creates a new GitHub release tag for the new beta version. Extension packages that also use CumulusCI require this release tag to find the latest version when this repository is listed as a dependency.
+* :ref:`Generates Release Notes <github_release_notes>`.
+* Syncs feature branches with the ``main`` branch, which automatically integrates the latest changes from ``main``. For more information see :ref:`auto merging`.
+
+.. tip:: 
+
+    To list each step in the ``upload_2gp_beta`` flow, run ``cci flow info upload_2gp_beta``.
+
+Customizing Package Uploads
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+2GP package uploads are performed by the ``create_package_version`` task. If the built-in configuration used by ``upload_2gp_beta``
+does not suit the needs of your project - for example, if you want to increment version
+numbers differently, or build an org-dependent package - you can customize the options for that task in ``upload_2gp_beta`` or invoke the task directly.
+
+To learn more about the available options, run
+
+.. code-block:: console
+
+    $ cci task info create_package_version
+
+
+CumulusCI can also create org-dependent and skip-validation packages when configured with the appropriate options.
```

Comment:
The org-dependent option is only available for Unlocked Packages, not managed 2gp